core/src/main/java/org/apache/spark/h2o/JavaH2OContext.java

```diff
@@ -45,16 +48,16 @@
  * can be set in H2O configuration class H2OConf
  *
  */
-public class JavaH2OContext {
+public class JavaH2OContext implements Serializable {
```

Comment:
Sorry, i am still confused why JavaH2OContext needs to be Serializable - it should live only on client, never send via wire.